includes/import.form.inc

```diff
@@ -390,11 +397,12 @@ function islandora_multi_importer_zip_plupload_form($form, &$form_state) {
  */
 function islandora_multi_importer_plupload_element_validate($element, &$form_state) {
   $saved_file = 0; // means no fid;
-  $scheme = file_default_scheme() . '://';
+  module_load_include('inc', 'islandora_multi_importer', 'includes/utilities');
+  $scheme = _imi_build_file_storage_directory();
   foreach ($form_state['values']['last']['file2'] as $uploaded_file) {
     if ($uploaded_file['status'] == 'done') {
       $source = $uploaded_file['tmppath'];
-      $destination = file_stream_wrapper_uri_normalize($scheme . "imi_".time()."_".$uploaded_file['name']);
+      $destination = file_stream_wrapper_uri_normalize($scheme . "/imi_".time()."_".$uploaded_file['name']);
```

Comment:
I see an extra / there... what if the location is the default scheme? or goes directly into private:// without a path?